src/Controller/BaseController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Controller;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Routing\Annotation\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+//todo: timestamp & blameable for entities;
+//todo: pagination
+//todo: sorting in templates
+//todo: "only private" filter in templates
+//todo: proper workout & playlist creation
+
+class BaseController extends AbstractController
+{
+
+    /**
+     * @param string $message
+     * @return void
+     */
+    public function addNoticeFlash($message)
+    {
+        $this->addFlash('notice', $message);
+    }
+    public function addSuccessFlash($message)
+    {
+        $this->addFlash('success', $message);
+    }
```

Comment:
NL between methods
Does your cs check work on this?

Reply:
cs doesn't complain about it, but I made the changes nevertheless.